.github/workflows/release.yaml

```diff
@@ -0,0 +1,243 @@
+name: Publish release
+
+env:
+  GRADLE_OPTS: '-Dorg.gradle.daemon=false -Dkotlin.compiler.execution.strategy=in-process'
+  JAVA_VERSION: 11
+  NODEJS_VERSION: 14
+  PROTOBUF_VERSION: 3.10.1
+  BINTRAY_API_USER: ${{ secrets.BINTRAY_API_USER }}
+  BINTRAY_API_KEY: ${{ secrets.BINTRAY_API_KEY }}
+
+on:
+  release:
+    types:
+      - created
```

Comment:
Did you intentionally change the trigger from using git tags (as in the CircleCI files) to using GitHub releases? I'm wondering why we wouldn't want to continue triggering from git tags?

Reply:
I had changed it to what felt like "the Github actions way of doing things", but I've changed it to use tags instead.